zerver/lib/email_mirror.py

```diff
@@ -161,12 +162,23 @@ class ZulipEmailForwardUserError(ZulipEmailForwardError):
     pass
 
 def send_zulip(sender: str, stream: Stream, topic: str, content: str) -> None:
+    trunc_topic = truncate_topic(topic)
+    if len(trunc_topic) < len(topic):
+        pdlte = datetime.datetime.today()
```

Comment:
We usually do ``from django.utils.timezone import now as timezone_now`` and use that, so it's more consistent to stick with that